tgui/packages/tgui/interfaces/Experimentator.tsx

```diff
@@ -0,0 +1,249 @@
+import { toTitleCase } from 'common/string';
+
+import { useBackend } from '../backend';
+import {
+  Button,
+  Icon,
+  Image,
+  LabeledList,
+  NoticeBox,
+  Section,
+  Stack,
+  Tooltip,
+} from '../components';
+import { Window } from '../layouts';
+
+type Data = {
+  hasItem: boolean;
+  isOnCooldown: boolean;
+  isServerConnected: boolean;
+  loadedItem: Item;
+};
+
+type Item = {
+  name: string;
+  icon: string;
+  isRelic: boolean;
+  associatedNodes: Node[];
+};
+
+type Node = {
+  name: string;
+  isUnlocked: boolean;
+};
```

Comment:
```suggestion
type Data = {
  hasItem: BooleanLike;
  isOnCooldown: BooleanLike;
  isServerConnected: BooleanLike;
  loadedItem: Item;
};

type Item = {
  name: string;
  icon: string;
  isRelic: BooleanLike;
  associatedNodes: Node[];
};

type Node = {
  name: string;
  isUnlocked: BooleanLike;
};
```
booleans passed from byond are 1 / 0, so we use BooleanLike (`number | boolean | null | undefined`) rather than boolean